src/api/handlers/guild.ts

```diff
@@ -445,6 +446,28 @@ export function fetchMembers(guild: Guild, options?: FetchMembersOptions) {
   }) as Promise<Collection<string, Member>>;
 }
 
+/**
+ * ⚠️ BEGINNER DEVS!! YOU SHOULD ALMOST NEVER NEED THIS AND YOU CAN GET FROM cache.members.get()
+ *
+ * ADVANCED:
+ * Highly recommended to **NOT** use this function to get members instead use fetchMembers().
+ * REST(this function): 50/s global(across all shards) rate limit with ALL requests this included
+ * GW(fetchMembers): 120/m(PER shard) rate limit. Meaning if you have 8 shards your limit is 960/m.
+*/
+export function getMembers(
+  guildID: string,
+  options?: GetMemberOptions,
+) {
+  if (!(identifyPayload.intents && Intents.GUILD_MEMBERS)) {
+    throw new Error(Errors.MISSING_INTENT_GUILD_MEMBERS);
+  }
+
+  return RequestManager.get(
```

Comment:
1. Create the structures.
2. Add to cache
3. Return the full member structs
4. This function should really support pagination by default